main.py

```
@@ -3,113 +3,165 @@
+GITHUB_PAGES = 3  # Number of GitHub pages to fetch per year
 
-# Database setup
+# -------------------- LOGGING --------------------
+logging.basicConfig(level=logging.INFO, format="%(asctime)s [%(levelname)s] %(message)s")
+
+# -------------------- DATABASE --------------------
 db = SqliteDatabase(DB_FILE)
 
 class CVE_DB(Model):
-    id = IntegerField()
+    id = IntegerField(unique=True)
```

Ruff (0.12.2)

`IntegerField` may be undefined, or defined from star imports

(F405)